pages/experience.tsx

```diff
@@ -30,29 +24,56 @@ const LoadingView = (): JSX.Element => (
   </main>
 );
 
-function Experience({ id }): JSX.Element {
+function Experience({ userId, dataId, isApparatusId }): JSX.Element {
   const [loading, setLoading] = useState(true);
-  const [jsonFile, setJsonFile] = useState([]);
+  const [experienceData, setExperienceData] = useState<ExperienceData>();
+  // either apparatusID is provided or experience id but not both
+
+  const experience: ExperienceData = {
+    apparatusMetadata: { Paths: [], Data: [] },
+    apparatusId: "",
+    initializationData: { actionList: [] },
+  };
 
```

Comment:
is it possible to annotate, for example userId the type of it.

Reply:
from annotation you mean defining the type of each variable?